SceneKitReel/SceneKitReelMac/SceneKitReelMac.csproj

```diff
@@ -33,7 +33,6 @@
     <AOTMode>None</AOTMode>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>full</DebugType>
```

Comment:
Hi @mykyta-bondarenko-gl - curious why you've removed this node throughout?

Reply:
Hi @bradumbaugh. VSfM did it after opening `Info.plist` in editor an I thought that it's ok if IDE fixes manifest too.

Reply:
Thanks. Tested it out quickly and everything looks good.